game.h

```diff
@@ -126,7 +126,6 @@ class game
 
   ///Manages collisons with walls
   player resolve_wall_collision(player p);
-
```

Comment:
Why did you remove this line?

Reply:
I don't know. Will revert.

Reply:
Oops, no I haven't! Sorry, shouldn't promise stuff before actually committing the changes 😅

Reply:
done now!